README.md

```diff
@@ -1 +1,92 @@
-# math-magic-react-setup
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To do list
+
+> Math-magic
+This is a Single Page Application (SPA) that allows users to Make simple calculations and Read a random math-related quote.
+
+# WORK FLOW SCREEN
+## SCREEN SHOT
+![screenshot](./)
+
+## Built With
+
+- REACT
+- CSS 3
+- JAVASCRIPT ES6
+
+## Live Demo (if available)
+
+[Live Demo Link]
+
+## Getting Started
+
+**Instructions on how to view and clone my project locally.**
+ To Access my project locally please follow the following instructions;
+1. Create a folder on your computer where you will clone the project
+2. Enter this url: https://github.com/dialite/math-magic-react-setup.git in your web browser.
+3. Once opened, move your cursor to the top left of the project page to select 'code' a green code button will be visible on the righthand corner showing download options.
+5. Select download Zip option from drop down menu to your desired folder on your computer.
+6. Once the download is complete you will be able to access my project locally.
+7. Also, you can navigate to your folder using a console.
+8. Then run git clone https://github.com/dialite/math-magic-react-setup.git
+
+### VScode, GIT bash
+
+### clone the repository to your local machine
+
+To get a local copy and running follow these simple example steps.
+
+- Set up a new repository and prepare it for development using best practices (e.g. linters).
+- Create a new branch according to the GitHub flow rules.
+- Add a .gitignore file.
+- Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+- Enable GH web Page on portfolio repository to deploy it on the web
+- Add descriptive README file to your project
+
+### Install
+
+- Git (as version control)
+- sudo apt install nodejs (To install nodejs)
+- npm install --save-dev jest (To get started with Jest)
+- npm install --save-dev jest-environment-jsdom(To get started with the JSDOM test environment)
+
+### Usage
+
+### Run tests
+
+- Lighthouse
+- Stylelint
+- ESLint
+- clear all Item/task (with Jest)
+- edit Item/task (with Jest)
+
+### Deployment
+
+## Authors
+
+👤 Raymond Olisa
+
+- GitHub: [@dialite](https://github.com/dialite)
+- Twitter: [@raymondolisa01](https://twitter.com/raymondolisa01)
+- LinkedIn: [raymond-olisa](https://www.linkedin.com/in/raymond-olisa-775929243/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Your MIT link is not working 😨 Kindly fix this link as shown below ⬇️

**Yours 🔴**
`[MIT](./MIT.md)`
**Correct 🟢**
`[MIT](./LICENSE)`